sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 2.3.5-beta.1 (Unreleased)
-
+## 2.3.5 (2020-09-14)
+_Bug Fixes_
```

Comment:
We follow the heading pattern `### Key Bug Fixes` in other changelogs.
Here and other places